services/workers/src/test/java/org/sagebionetworks/table/worker/TableWorkerIntegrationTest.java

```diff
@@ -2798,6 +2801,57 @@ public void testPLFM_5240Open() throws Exception {
 		});
 	}
 
+	/**
+	 * PLFM-6392
+	 * @throws Exception
+	 */
+	@Test
+	public void testQueryWithUnquotedKeyword() throws Exception{
+		ColumnModel year = new ColumnModel();
+		year.setColumnType(ColumnType.STRING);
+		year.setMaximumSize(50L);
+		year.setName("year");
+		year = columnManager.createColumnModel(adminUserInfo, year);
+		schema = Lists.newArrayList(year);
+		// build a table with this column.
+		createTableWithSchema();
+		TableStatus status = waitForTableProcessing(tableId);
+		assertTrue(TableState.AVAILABLE.equals(status.getState()));
+
+		RowSet rowSet = new RowSet();
+		rowSet.setRows(Lists.newArrayList(
+				TableModelTestUtils.createRow(null, null, "2020")));
+		rowSet.setHeaders(TableModelUtils.getSelectColumns(schema));
+		rowSet.setTableId(tableId);
+		referenceSet = appendRows(adminUserInfo, tableId,
+				rowSet, mockProgressCallback);
+
+		Throwable selectClauseException = assertThrows(IllegalArgumentException.class, ()->{
+			waitForConsistentQuery(adminUserInfo, "select year from " + tableId, null, null, (queryResult) -> {
+				fail("Throw IllegalArgumentException");
+			});
+		});
+		assertNotNull(selectClauseException);

```

Comment:
The null check is not needed as the assertThrows always returns the exception or fail